src/components/VideoChatButtonAndMenu/BaseVideoChatButtonAndMenu.js

```diff
@@ -28,131 +28,119 @@ const propTypes = {
     ...windowDimensionsPropTypes,
 };
 
-class BaseVideoChatButtonAndMenu extends Component {
-    constructor(props) {
-        super(props);
+function BaseVideoChatButtonAndMenu(props) {
+    const [isVideoChatMenuActive, setMenuVisibility] = useState(false);
```

Comment:
We could name anything for the setter function, but this will lead to confusion between the state name, and I didn't find in our code name the function differently from the state.

```suggestion
    const [isVideoChatMenuActive, setIsVideoChatMenuActive] = useState(false);
```